pages/createUpdate.js

```diff
@@ -89,7 +91,13 @@ class CreateUpdatePage extends React.Component {
       if (update.isChangelog) {
         update.isPrivate = false;
       }
-      const res = await this.props.createUpdate({ variables: { update } });
+      const res = await this.props.createUpdate({
+        variables: { update },
+        refetchQueries: [
+          { query: updatesQuery, context: API_V2_CONTEXT, variables: { collectiveSlug: this.props.slug } },
+          { query: updatesSectionQuery, variables: { slug: this.props.slug, onlyPublishedUpdates: true } },
```

Comment:
You need to provide the same variables for `refetchQuery` to do its job. I'll let you check, but you should have something like:

```suggestion
          { query: updatesQuery, context: API_V2_CONTEXT, variables: { collectiveSlug: this.props.slug, offset: 0, limit: UPDATES_PER_PAGE, includeHostedCollectives: false  } },
          { query: updatesSectionQuery, variables: getUpdatesSectionQueryVariables(this.props.slug) } },
```

Reply:
@Betree : Thanks. You are right. That makes sense. 👍🏽